Xamarin.Forms.Maps.WPF/Xamarin.Forms.Maps.WPF.csproj

```diff
@@ -1,99 +1,20 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{89B0DB73-A32E-447C-9390-A2A59D89B2E4}</ProjectGuid>
-    <OutputType>library</OutputType>
-    <RootNamespace>Xamarin.Forms.Maps.WPF</RootNamespace>
-    <AssemblyName>Xamarin.Forms.Maps.WPF</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <ProjectTypeGuids>{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <WarningLevel>4</WarningLevel>
-    <TargetFrameworkProfile />
-    <!-- 
-        This is needed for non SDK style project so the IDE will pick the correct LangVersion.
-        Once this is an SDK style project it will pick this up from the Directory.Build.Props file correctly
-     -->
-    <LangVersion>7.3</LangVersion>
+    <TargetFramework>net461</TargetFramework>
```

Comment:
we have a dependency on System.Device which is not ported to .NET Core. so build only net461